deploy/operator.yaml

```diff
@@ -3183,7 +3183,7 @@ metadata:
   namespace: openebs
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
```

Comment:
I will add that note in the upgrade doc in openebs/upgrade repo.

Reply:
Having a Role in the cluster will not cause any problem and it is a one-time activity of deleting when user upgrading to later version so we can add in the document  instead of handling it in code